src/api/stats/getChainTvl.js

```diff
@@ -25,11 +26,12 @@ const getChainTvl = async chain => {
     const vaultBal = vaultBalances[i];
     let tokenPrice = 0;
     try {
+      // tokenPrice = 15.5;
       tokenPrice = await fetchPrice({ oracle: vault.oracle, id: vault.oracleId });
     } catch (e) {
       console.error('getTvl fetchPrice', chainId, vault.oracle, vault.oracleId, e);
     }
-    const tvl = vaultBal.times(tokenPrice).dividedBy(10 ** (vault.tokenDecimals ?? 18));
+    const tvl = vaultBal.times(tokenPrice).shiftedBy(vault.tokenDecimals ?? 18);
```

Comment:
not 100% certain but I think this should be shiftedby a negative number to get the same result

Reply:
yeah should be negative 👍